airflow/cli/commands/task_command.py

```diff
@@ -174,6 +175,12 @@ def _run_raw_task(args, ti: TaskInstance) -> None:
     )
 
 
+def _extract_external_executor_id(args) -> Optional[str]:
+    if "external_executor_id" in args:
+        return args.external_executor_id
+    return os.environ.get("external_executor_id", None)
```

Comment:
Looks like we didn't set `external_executor_id` in the environment, am I missing something?